fluent-syntax/src/parser.js

```diff
@@ -394,6 +395,17 @@ export default class FluentParser {
     return new AST.NumberLiteral(num);
   }
 
+  getValue(ps) {
+    let pattern = this.getPattern(ps);
+
+    if (pattern.elements.length === 1 &&
+        pattern.elements[0].type === "Placeable" &&
+        pattern.elements[0].expression.type === "VariantList") {
+      return pattern.elements[0].expression;
```

Comment:
Could we instead remove the following code from `getExpression` and move it up here? 

https://github.com/projectfluent/fluent.js/blob/108ed2438fa37416f38f5786323f4b89d2d38d58/fluent-syntax/src/parser.js#L542-L548

`getPattern` should not be able to successfully parse selector-less `SelectExpressions`.

Reply:
We'll also need to change `getVariant` to use `getValue` rather than `getPattern`:

https://github.com/projectfluent/fluent.js/blob/108ed2438fa37416f38f5786323f4b89d2d38d58/fluent-syntax/src/parser.js#L382

And then, in `getSelectorExpression` ensure that none of the parsed variants has a `VariantList` for value:

https://github.com/projectfluent/fluent.js/blob/108ed2438fa37416f38f5786323f4b89d2d38d58/fluent-syntax/src/parser.js#L582